### PR TITLE
[INLONG-10638][Manager] Data preview supports filtering function

### LINKED ISSUE #10638: [Improve][Manager] Data preview supports filtering function
### Description

Data preview supports filtering function.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #10638

### Motivation

Data preview supports filtering function.

### Modifications

Data preview supports filtering function.
1.Add `QueryMessageRequest`, Including `FieldName`、 `operationType`、`targetValue`.
2.When data preview needs to be filtered, all three parameters above must be set.
3.`FieldName` is used to set the field as the filtering condition, and `operationType` is used as the filtering operation type, such as `like` 、`=`、`!=` , `TargetValue` is a comparative value.

### CODE REVIEW

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongStreamClient.java [hunk 1]
```diff
@@ -295,11 +300,15 @@ public List<StreamField> parseFields(String method, String statement) {
         return parseFields(request);
     }
 
-    public List<BriefMQMessage> queryMessage(String groupId, String streamId, Integer messageCount) {
-        Preconditions.expectNotBlank(groupId, ErrorCodeEnum.GROUP_ID_IS_EMPTY);
-        Preconditions.expectNotBlank(streamId, ErrorCodeEnum.STREAM_ID_IS_EMPTY);
+    public List<BriefMQMessage> queryMessage(QueryMessageRequest request) {
+        Preconditions.expectNotBlank(request.getGroupId(), ErrorCodeEnum.GROUP_ID_IS_EMPTY);
+        Preconditions.expectNotBlank(request.getStreamId(), ErrorCodeEnum.STREAM_ID_IS_EMPTY);
+        Map<String, Object> requestMap = JsonUtils.OBJECT_MAPPER.convertValue(request,
```
Comment (vernedeng) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongStreamClient.java:306]: JsonUtils.parseObject

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/InlongMsgDeserializeOperator.java [hunk 2]
```diff
@@ -87,6 +86,10 @@ public List<BriefMQMessage> decodeMsg(InlongStreamInfo streamInfo, byte[] msgByt
                     DataTypeOperator dataTypeOperator =
                             dataTypeOperatorFactory.getInstance(DataTypeEnum.forType(streamInfo.getDataType()));
                     List<FieldInfo> streamFieldList = dataTypeOperator.parseFields(body, streamInfo);
+                    boolean ifFilter = checkIfFilter(request, streamFieldList);
+                    if (!ifFilter) {
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/InlongMsgDeserializeOperator.java:90]:
```suggestion
if (!checkIfFilter(request, streamFieldList))
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/RawMsgDeserializeOperator.java [hunk 3]
```diff
@@ -59,6 +60,10 @@ public List<BriefMQMessage> decodeMsg(InlongStreamInfo streamInfo,
             DataTypeOperator dataTypeOperator =
                     dataTypeOperatorFactory.getInstance(DataTypeEnum.forType(streamInfo.getDataType()));
             List<FieldInfo> fieldList = dataTypeOperator.parseFields(body, streamInfo);
+            boolean ifFilter = checkIfFilter(request, fieldList);
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/RawMsgDeserializeOperator.java:63]: ditto

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/InlongMsgDeserializeOperator.java (hunk 2 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/InlongMsgDeserializeOperator.java]: Fixed.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/RawMsgDeserializeOperator.java (hunk 3 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/RawMsgDeserializeOperator.java]: Fixed.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongStreamClient.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongStreamClient.java:306]: The purpose here is to convert the object into a map, which `JsonUtils.parseObject` cannot achieve.

**Review by aloyszhang — DISMISSED** [review]

**Review by aloyszhang — APPROVED** [review]

**Review by vernedeng — DISMISSED** [review]

**Review by aloyszhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Pull request merged**